js/post-select/components/post-list-item.js

```diff
@@ -14,7 +15,7 @@ const PostListItem = ( { post, author, thumbnail, postTypeObject, isSelected, on
 		<label htmlFor={ `select-post-${post.id}` }>
 			{ thumbnail
 				? <img
-					alt={ post.title.rendered }
+					alt={ DOMPurify.sanitize( post.title.rendered ) }
```

Comment:
Nitpick, better to `import { sanitize } from DOMPurify;` and then call these as `alt={ sanitize( post.title.rendered ) }`

(Although, I'm confused why we need to do this -- React should already be escaping the alt attribute. The main security issue with React is when you set `__dangerouslySetInnerHtml`. Are we certain we need to double-sanitize alt strings?)

Reply:
@kadamwhite thanks for the review.

I applied the solution you suggested on both, but could not yet test it, as I'm setting up a local environment to use the plugin.

I got this solution of sanitizing the variable and using `DOMPurify` from the security audit:
https://semgrep.dev/r?q=typescript.react.security.audit.react-dangerouslysetinnerhtml.react-dangerouslysetinnerhtml